splunk.json

```diff
@@ -389,7 +372,7 @@
         {
             "action": "on poll",
             "description": "Ingest logs from the Splunk instance",
-            "verbose": "The configured query is what will be used during ingestion. If you only wish to show certain fields, then you can specify these as a comma-separated list in the configuration. If left unspecified, all available fields will be added to each artifact. When limiting the number of events to ingest, it will ingest the most recent events.<br><br>",
+            "verbose": "The configured query is what will be used during ingestion. If you only wish to show certain fields, you can specify these as a comma-separated list in the configuration. If left unspecified, all available fields will be added to each artifact. When limiting the number of events to ingest, it will ingest the most recent events. To avoid duplication in polling, append | fields * to the query.<br><br>",
```

Comment:
I don't understand the added text

Reply:
it says that while polling, if the user wants that only some specific fields from the data into the artifact, then they need to be mentioned in the asset config param, otherwise all the  fields will be included in the artifacts.

Reply:
I meant what exactly has to be appended? " | fields *"? Can you add quotes or something to make this clear?